uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/main.js

```diff
@@ -37,10 +37,6 @@ define([
 
         modalInstance.result.then(function (results) {
           $scope.$broadcast('documentFormSuccess', results, data);
-
-          if (results.open) {
```

Comment:
The above function is needed to open a document, if ` $scope.document.open` was set true. But, Since we have removed "Save and New" button as per the wireframe, we wont be needing to  pen new modal once the document is saved. So, we won't be needing it any more.